### PR TITLE
task(issue-95): create shipping page, populate tables, and update status

### PR DESCRIPTION
**Pull request by mlee97**
User with shipping access rights now have access to a page called "Shipping".

Screenshot 2021-03-14 211803

The shipping page is populated by the data fetched from the Orders table.

Screenshot 2021-03-14 211902

The status of an order can be toggled by pressing the button situated on its respective row.

Screenshot 2021-03-14 211921

### CODE REVIEW

File: ERP/routes/web.php [hunk 1]
```diff
@@ -121,3 +122,9 @@
     Route::get('update-bike/{id}', [BikeController::class, 'updateBike'])
         ->name('update.bike');
 });
+
+//Shipping Routes given `shipping.access.only` middleware (prevents non-shipping or non-IT personal from accessing this route)
```
Comment (CeliaCaii) [inline ERP/routes/web.php:130]: IT personal can access the Shipping page?

Comment (mlee97) [reply ERP/routes/web.php:130]: What's the super-user now?

Comment (CeliaCaii) [reply ERP/routes/web.php:130]: We technically don't have a super user yet

Comment (mlee97) [reply ERP/routes/web.php:130]: ok i will take it out for you.

**Review by CeliaCaii — APPROVED** [review]
Good work!

**Review by pirasunt — APPROVED** [review]

**Pull request merged**